### ISSUE TITLE
Error in make prax

### ISSUE THREAD

**Issue by rpbaptist** [ysbaddaden/prax.cr#20]
When I try to make prax I get the following error:

```
$ make                          
mkdir -p bin
/usr/bin/crystal build /home/richard/code/prax.cr/src/prax.cr -o bin/prax-binary
Error in ./src/event.cr:10: undefined method 'dns_base' for Event::Base
```

I am not sure how to troubleshoot this.

**Comment by ysbaddaden** [comment]
Crystal 0.9.1 fixed a problem I manually fixed in 3b91e60bee5f577c1985bae525a83436b73dc708 which I must revert. Crystal 0.9.1 [has another bug](https://github.com/manastech/crystal/pull/1856), so master is required, and I have some other changes to integrate.

**Comment by ysbaddaden** [comment]
Prax (master) should now be compatible with Crystal (master)!
